xero_files.yaml

```diff
@@ -518,6 +518,42 @@ paths:
                               "ObjectGroup":"Contact"
                           }
                         ]'
+  /Associations/Count:
+    parameters:
+      - $ref: '#/components/parameters/requiredHeader'
+    get:
+      security:
+        - OAuth2: [files, files.read]
+      tags:
+        - Files
+      summary: Retrieves a count of associations for a list of objects.
+      operationId: getAssociationsCount
+      description: By passing in the appropriate options, 
+      parameters: 
+        - name: ObjectIds
+          in: query
+          required: true
+          description: A comma-separated list of object ids
+          schema: 
+            type: array
+            items:
+              type: string
+              format: uuid
+              example: 1270bf7c-5d18-473a-9231-1e36c4bd33ed
+      responses:
+        '200':
+          description: A dictionary of the object Ids and associations count
+          content:
+            application/json:
+              schema:
+                type: array
```

Comment:
@eduardocopat-xero hey, think this needs to be response type 'object'. Please can you update?

Reply:
@pumpkinball Submitted a new commit fixing this. Good catch,